stable/spinnaker/requirements.yaml

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: redis
-  version: 3.8.0
+  version: 10.5.3
```

Comment:
Since this is a massive chart change, did you address the possibility of any core changes in the redis chart? There might have been some variables moved round.

Reply:
With the default in values.yaml (which is all I'm using) the redis service/pod deploys without issue.